app/views/properties/show.html.erb

```diff
@@ -0,0 +1,38 @@
+<%= form_for([ @property, @reservation ]) do |f| %>
```

Comment:
Just use @reservation here since you will be saving for reservation instead of @property